libs/financial-aid/shared/src/components/FileList/FileList.tsx

```diff
@@ -71,7 +72,11 @@ const FileList = ({ className, files }: Props) => {
                   item,
                 )} KB`}</Text>
                 {'created' in item && (
-                  <Text variant="small"> {`${item.created}`}</Text>
+                  <div className={styles.date}>
+                    <Text variant="small">
+                      {format(new Date(item.created), 'dd.MM.y  · kk:mm')}
```

Comment:
Not to be addressed in this pr but maybe we should have a date format funtion in utils. It would help if we do ever change the formats then we just have to do it in a single place.